### ISSUE TITLE
hang in cm/test_uct_stats.flush_wait_iface/2

### ISSUE THREAD

**Issue by yosefe** [openucx/ucx#965]
http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2367/console
w/o valgrind

```
13:59:39 [ RUN      ] cm/test_uct_stats.flush_wait_iface/2
13:59:46 [1473677986.628758] [jenkins03:2529 :3]       cm_iface.c:210  UCX  ERROR SIDR request error, status: response timeout error
13:59:46 [1473677986.628793] [jenkins03:2529 :3]       cm_iface.c:210  UCX  ERROR SIDR request error, status: response timeout error
```

**Comment by yosefe** [comment]
similar failure:
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1715/label=hpc-test-node/console

```
02:09:26 [ RUN      ] cm/uct_flush_test.am_pending_flush_nb/0
02:09:26 [1473721766.862912] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862928] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862943] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862955] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862970] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862981] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.862992] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.863002] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.863012] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 [1473721766.863024] [hpc-test-node:6791 :0]      uct_iface.c:51   UCX  WARN  got active message id 1, but no handler installed
02:09:26 /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node/contrib/../test/gtest/common/test.cc:133: Failure
02:09:26 Failed
02:09:26 Got 10 warnings during the test
02:09:26 
[  FAILED  ] cm/uct_flush_test.am_pending_flush_nb/0, where GetParam() = cm/mlx5_0:1 (180 ms)
02:09:26 [ RUN      ] cm/uct_flush_test.am_flush_ep_no_comp/0
```

**Comment by yosefe** [comment]
fixed in #1088

**Comment by yosefe** [comment]
still happened in http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2119/label=hpc-test-node/consoleFull
